code/modules/instruments/instrument_data/fun.dm

```diff
@@ -23,3 +23,8 @@
 	id = "mothscream"
 	real_samples = list("60"='sound/voice/moth/scream_moth.ogg')
 	admin_only = TRUE
+
+/datum/instrument/fun/bilehorn
```

Comment:
need to set admin_only to TRUE so the synthesizer cant use it

Reply:
what a terrible var name